tex/gregoriotex-spaces.tex

```diff
@@ -384,6 +384,15 @@
 
 % dimen keeping the shift computed with next function
 \newdimen\gre@dimen@eolshift
+\newcount\gre@count@protrusion@hyphen@eol %
+
+% protrusion for hyphens inerted by Lua
+\gre@count@protrusion@hyphen@eol = 50
```

Comment:
Just for my own edification, we normally put `\relax` after a number to for safety.  Why not here?